Src/BootCamp.Chapter/Lesson3.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace BootCamp.Chapter
+{
+    class Lesson3
+    {
+        static string name = GetName();
```

Comment:
GetX is a fine name, but you don't just get a name, you also request for it. PromptX sounds better, because it refers to requesting in console and getting it.